nsis/src/nsis/NsisAnalyzer.java

```diff
@@ -75,11 +81,86 @@ public boolean added(Program program, AddressSetView set, TaskMonitor monitor, M
 			throws CancelledException {
 		MemoryBlock entriesBlock = program.getMemory()
 				.getBlock(NsisConstants.ENTRIES_MEMORY_BLOCK_NAME);
-		Disassembler disassembler = Disassembler.getDisassembler(program, monitor, null);
-		AddressSet modifiedAddrSet = disassembler.disassemble(entriesBlock.getStart(), null);
+		AddressSet modifiedAddrSet = disassembleByteCode(program, entriesBlock, monitor);
+
 		if (modifiedAddrSet.isEmpty()) {
 			return false;
 		}
+
+		MemoryBlock stringsBlock = program.getMemory()
+				.getBlock(NsisConstants.STRINGS_MEMORY_BLOCK_NAME);
+		InstructionIterator instructions = program.getListing().getInstructions(modifiedAddrSet,
+				/* forward direction */ true);
+
+		for (Instruction instr : instructions) {
+			try {
+				resolveStrings(instr, stringsBlock);
+			} catch (MemoryAccessException e) {
+				monitor.setMessage(
+						"Unable to revolve strings at instruction: " + instr.getAddressString(
```

Comment:
This spacing looks odd, is that what Eclipse is doing automatically?

Reply:
yes

Reply:
I guess that's too late to notice, but Google's style guide is "2 spaces indent", and here I see all 8.

Not sure it's worth changing it now, but 8-spaces indent makes it hard to respect 100 chars limit

Reply:
I could change it and open a PR just for that. Would it be worth it?

Reply:
Separate PR sounds good to me.

Reply:
Also see https://github.com/google/google-java-format

Reply:
There is a way to have Eclipse enforce this for you. I'm having trouble remembering if github CI could be used to also enforce it.